src/main/java/uk/gov/ons/ctp/response/casesvc/CaseSvcApplication.java

```diff
@@ -90,12 +88,10 @@ public CustomSpringLiquibase liquibase() {
     LiquibaseProperties liquibaseProperties = liquibaseProperties();
     SpringLiquibase liquibase = new SpringLiquibase();
     liquibase.setChangeLog(liquibaseProperties.getChangeLog());
-    liquibase.setContexts(liquibaseProperties.getContexts());
```

Comment:
No longer required as we don't use multiple contexts in our migrations. The contexts have changed from a list of strings to a comma separated strings. But as we don't use contexts there was no point in handling this.